src/forstudents-whyjoin/WhyJoin.css

```diff
@@ -6,33 +6,38 @@
 .whyjoin-text {
   color: var(--clr-hl1);
   text-align: right;
-  font-size: 36px;
+  font-size: 32px;
 }
 
 .whyjoin-text > h2 {
   font-size: 72px;
 }
+
+@media (max-width: 480px) {
+  .whyjoin-text > p {
+    text-align: justify;
```

Comment:
Hey Han, I noticed that having this property makes the text have uneven spacing (in mobile view), and removing it seemed to fixed the issue.